core_backend/app/question_answer/routers.py

```diff
@@ -155,9 +156,11 @@ async def stt_llm_response(
         os.remove(file_path)
         file_stream.close()
 
-    if type(response) is QueryResponse:
+    if isinstance(response, AudioResponse):
         return response
-    elif type(response) is QueryResponseError:
+    elif isinstance(response, QueryResponse):
+        return response
+    elif isinstance(response, QueryResponseError):
```

Comment:
are you sure isinstance() works here? It didn't work for /search which is why I used the if type() is XX format.

Reply:
oh it didn't? atleast while testing it was working, i was using the type( ) only but during the hurdle sid pointed it out that instead of type we should use isinstance cause that is better